File: docs/components_page/components/input/select.jl
using DashBootstrapComponents

select = dbc_select(
    id = "select",
    options = [
        Dict("label" => "Option 1", "value" => "1"),
        Dict("label" => "Option 2", "value" => "2"),
        Dict("label" => "Disabled option", "value" => "3", "disabled" => true),
    ],
);

File: docs/components_page/components/input/select.R
library(dashBootstrapComponents)

select <- dbcSelect(
  id = "select",
  options = list(
    list(label = "Option 1", value = "1"),
    list(label = "Option 2", value = "2"),
    list(label = "Disabled option", value = "3", disabled = TRUE)
  )
)
